bin/coreos-install

```diff
@@ -334,6 +343,27 @@ if [[ -n "${CLOUDINIT}" ]] || [[ -n "${COPY_NET}" ]]; then
     umount "${WORKDIR}/rootfs"
 fi
 
+if [[ -n "${IGNITION}" ]]; then
+  # The OEM partition should be #3 but make no assumptions here!
+  # Also don't mount by label directly in case other devices conflict.
+  OEM_DEV=$(blkid -t "LABEL=OEM" -o device "${DEVICE}"*)
+
+  if [[ -z "${OEM_DEV}" ]]; then
+    echo "Unable to find new OEM partition on ${DEVICE}" >&2
+    exit 1
```

Comment:
The EXIT trap needs to be updated to be just the rm command immediately after unmounting rootfs above. In the unlikely event of using both -c and -i we don't want the trap to erroneously try to unmont rootfs again.